no-yan/cache-actions-compare
# refactor/extractCacheCheck

Extracted check-cache that checks cache without download.
This is better when checking cache before many actions run in parallel. This makes checking cache cost almost free, but can reduce the number of cache-misses from n to 1. 

- set output
- delete npm related steps
- set output correctly, but they can be string?
- test
- test
- add doc
- expose cache-key as input
- add comment
- update npm.yaml to use cache-check
- create cache when cache miss
